### ISSUE TITLE
Créer un site pour OSM Data ?

### ISSUE THREAD

**Issue by datagistips** [data-osm/frontend#83]
- Nous contacter

**Comment by datagistips** [comment]
Les plus d'OSM Data, ses fonctionnalités,...

**Comment by TANK2003** [comment]
Il serait questions de blogs non ? 

En vrac j'ai des idées de blog pour expliquer : 

- La fiche descriptive d'un POI : la traduction des tags, mappilary, les horaires d'ouvertue, les icones d'acces rapide (facebook, site web, wiki, etc...), etc...
- Ajout d'une couche OSMdata dans QGIS : par flux ou en téléchargeant la donnée
- Ajout 'une couche OSMdata dans UMAP
- Créer une couche et un style sur OSMdata
- Le pipeline de création d'une couche dans OSM data : création du provider depuis la requète écrite par l'utilisateur >  la constitution de la vue dans la base de données >  l'ajout de la vue dans un projet QGIS >  application d'un style par défaut OSMData à la couche > visualisation de la couche grâce au WMS. Tout ceci avec des bribes de codes
- Les stats d'OSMdata, les couches les plus visualisées au fil du temps, le nombre de requetes WMS sur le serveur par jour, le nombre de téléchargements, 

qu'es ce que tu en penses ? tu peux completer, ensuite on voit comment le faire et où le publier.

**Comment by TANK2003** [comment]
lier à #82 (Renommer demo.openstreetmap.fr)

data.openstreetmap.fr
openstreetmap.data.fr
data.osm.fr
osm.data.fr
osm-data.fr

**Comment by datagistips** [comment]
Oui, je pense qu'un blog serait pas mal, avec dans un premier temps le cas d'usage le plus utile qui est selon moi l'ajout d'une couche issue d'OSMData dans QGIS (avec le style associé ;-) )

**Comment by datagistips** [comment]
On pourrait créer un site assez facilement en osmdata.github.io avec dedans des markdowns expliquant comment utiliser la plateforme :)
